docs/en/controller/quick_start.md

```diff
@@ -0,0 +1,200 @@
+# Master-Slave automatic switch Quick start 
+
+## Introduction
+
+![架构图](../image/controller/controller_design_2.png)
+
+This document mainly introduces how to quickly build a RocketMQ cluster that supports automatic master-slave switch, as shown in the above diagram. The main addition is the Controller component, which can be deployed independently or embedded in the NameServer.
+
+For detailed design ideas, please refer to [Design Ideas](https://chat.openai.com/chat/design.md).
+
+For detailed guidelines on new cluster deployment and old cluster upgrades, please refer to [Deployment Guide](https://chat.openai.com/chat/deploy.md).
+
+## Compile RocketMQ source code
+
+```shell
+$ git clone https://github.com/apache/rocketmq.git
+
+$ cd rocketmq
+
+$ mvn -Prelease-all -DskipTests clean install -U
+```
+
+## Quick deployment
+
+After successful build
+
+```shell
+#{rocketmq-version} replace with rocketmq actual version. example: 5.0.0-SNAPSHOT
+$ cd distribution/target/rocketmq-{rocketmq-version}/rocketmq-{rocketmq-version}/
+
+$ sh bin/controller/fast-try.sh start
+```
+
+If the above steps are successful, you can view the status of the Controller using the operation and maintenance command.
+
+```shell
+$ sh bin/mqadmin getControllerMetaData -a localhost:9878
+```
+
+`-a` represents the address of any controller in the cluster
+
+At this point, you can send and receive messages in the cluster and perform switch testing.
+
+If you need to shut down the quick cluster, you can execute：
```

Comment:
```suggestion
If you need to shutdown the cluster quickly, you can execute：
```

Reply:
@Oliverwqcwrw  I will fix it later, thanks for you  review

Reply:
@Oliverwqcwrw `shutdown` is a noun... `need to shut down the cluster quickly` is the correct way.

Reply:
@lizhanhui agree with